File: 4- Prefix Sum/C#/1- Find the Highest Altitude.cs
public class Solution {
    public int LargestAltitude(int[] gain) {
        int maxi = Math.Max(gain[0] , 0) ; 
        
        for (int i = 1 ; i < gain.Length ; i++){
            gain[i] += gain[i-1] ; 
            maxi = Math.Max(gain[i] , maxi );
        }
        return maxi ; 
    }
}

File: 04- Prefix Sum/1- Find the Highest Altitude.cpp
class Solution {
public:
    int largestAltitude(vector<int>& gain) {
        int maxi = max(gain[0], 0);
        for (int i = 1; i < gain.size(); i++) {
            gain[i] += gain[i - 1];
            maxi = max(gain[i], maxi);
        }
        return maxi;
    }
};
